typings/bash.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'bash';
```

Comment:
Does this work with `ts-node` v8.x (version introduced in v11-meta), when running `./bin/balena-dev`? My previous experience was that it was necessary to create a folder named after the module, something like `typings/bash/index.d.ts`. (According to changes introduced by the first PR to v11-meta.)

Reply:
Done.